src/main/java/com/playmonumenta/networkrelay/RemotePlayerManager.java

```diff
@@ -78,26 +90,19 @@ public void broadcast() {
 			remotePlayerData.addProperty("isHidden", mIsHidden);
 			remotePlayerData.addProperty("isOnline", mIsOnline);
 			remotePlayerData.addProperty("shard", mShard);
-
-			//Gather Plugin data
-			GatherRemotePlayerDataEvent event = new GatherRemotePlayerDataEvent();
-			Bukkit.getPluginManager().callEvent(event);
-			mPluginData.clear();
-			mPluginData.putAll(event.getPluginData());
-
-			remotePlayerData.add("plugins", deserializePluginData());
+			remotePlayerData.add("plugins", serializePluginData());
 
 			try {
-				NetworkRelayAPI.sendBroadcastMessage(RemotePlayerManager.REMOTE_PLAYER_UPDATE_CHANNEL, remotePlayerData);
+				NetworkRelayAPI.sendExpiringBroadcastMessage(REMOTE_PLAYER_UPDATE_CHANNEL,
+					remotePlayerData,
+					REMOTE_PLAYER_MESSAGE_TTL);
 			} catch (Exception e) {
-				RemotePlayerManager manager = RemotePlayerManager.getInstance();
-				if (manager != null) {
-					CustomLogger.getInstance().ifPresent((logger) -> logger.warning("Failed to broadcast to channel " + REMOTE_PLAYER_UPDATE_CHANNEL + " reason: " + e.getMessage()));
-				}
+				MMLog.severe("Failed to broadcast " + REMOTE_PLAYER_UPDATE_CHANNEL);
 			}
 		}
 	}
 
+	public static final int REMOTE_PLAYER_MESSAGE_TTL = 5;
```

Comment:
We don't want to keep these messages around for very long - if they're too far out of date, they should be discarded sooner. This should probably move to a config, but here works for now.